overview.tex

```diff
@@ -132,13 +132,13 @@ \subsection*{Implementation and architecture}
     Users have a command line interface to use out of the box,
     however one can also use the package in their own software.
 
-    Implementation is very straightforward. In fact, algorithm:
+    Implementation is very straightforward:
     \begin{enumerate}
-      \item parses the options passed as arguments, creating a
-        structure with all  information;
-      \item parses and process  input files, using  
-         performance function definition to create  data to be plotted;
-      \item uses the chosen backend to plot  data.
+      \item Parse the options passed as arguments, creating a
```

Comment:
I think this is the only place on the text that should be small caps, because the other 2 places are: 1. OPTIONS ou 2. Names of Columns. 
BTW, I merged mannually.